recipes/simfil/all/conanfile.py

```diff
@@ -0,0 +1,100 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+
+
+required_conan_version = ">=1.62.0"
+
+
+class SimfilRecipe(ConanFile):
+    name = "simfil"
+    description = "simfil is a C++ 17 library and a language for querying structured map feature data. The library provides an efficient in-memory storage pool for map data, optimized for the simfil query language, along with a query interpreter to query the actual data."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Klebert-Engineering/simfil"
+    license = "BSD 3-Clause"
+    topics = ["query language"]
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_json": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_json": True,
+    }
+
+    exports_sources = "include/*", "src/*"
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "16",
+            "msvc": "192",
+            "gcc": "10",
+            "clang": "10",
+            "apple-clang": "14",
+        }
+
```

Comment:
```suggestion

    def configure(self):
        if self.options.shared:
            self.options.rm_safe("fPIC")

```

The fPIC is always removed when building shared, because shared without fPIC is not used at all.